vllm/lora/models.py

```diff
@@ -541,6 +562,19 @@ def _match_target_modules(self, module_name: str):
                 module_name) or target_module == module_name
             for target_module in self.supported_lora_modules)
 
+    def _filter_unsupported_module(self, module_name: str) -> bool:
```

Comment:
```suggestion
    def _filter_unsupported_mm_module(self, module_name: str) -> bool:
```

Add `mm` to indicate that it's about the multi-modal modules.